docs/kustomize.md

```diff
@@ -66,3 +66,8 @@ specified in the kustomization configuration:
 
 ![Kustomize Preview Content](img/kustomize-preview-content.png)
 
+## Editing kustomizations and Helm Values
```

Comment:
this section should only talk about Kustomize, not Helm

Reply:
Resolved.